<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>887</width>
    <height>499</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QGroupBox" name="time_available_grpbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>341</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="time_layout_horiz">
    <item>
     <widget class="QLabel" name="time_available_lbl">
      <property name="text">
       <string>Time Available:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="available_from_time_edit"/>
    </item>
    <item>
     <widget class="QLabel" name="time_to_lbl">
      <property name="text">
       <string>to</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="available_to_time_edit"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="location_grpbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="location_layout_horiz">
    <item>
     <widget class="QLabel" name="location_lbl">
      <property name="text">
       <string>Location:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="location_comb">
      <item>
       <property name="text">
        <string>Sullivan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Petty</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="days_available_grpbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>611</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="location_layout_horiz_2">
    <item>
     <widget class="QLabel" name="days_available_lbl">
      <property name="text">
       <string>Days Available:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="all_days_chk">
      <property name="text">
       <string>All Days</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="monday_chk">
      <property name="text">
       <string>Monday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="tues_chk">
      <property name="text">
       <string>Tuesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="wed_chk">
      <property name="text">
       <string>Wednesday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="thurs_chk">
      <property name="text">
       <string>Thursday</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="fri_chk">
      <property name="text">
       <string>Friday</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="category_grpbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="category_layout_horiz">
    <item>
     <widget class="QLabel" name="categories_lbl">
      <property name="text">
       <string>Categories and Attributes:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="category_comb">
      <item>
       <property name="text">
        <string>WI</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GLT</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>SI</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>470</y>
     <width>241</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="professor_grpbox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="professor_layout_horiz">
    <item>
     <widget class="QLabel" name="professor_lbl">
      <property name="text">
       <string>Professor:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="professor_comb">
      <item>
       <property name="text">
        <string>Dr. Howie</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dr. Aleshire</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
